pkg/cloud/services/compute/instance.go

```diff
@@ -152,12 +174,13 @@ func createInstance(is *Service, eventObject runtime.Object, clusterName string,
 	accessIPv4 := ""
 	portList := []servers.Network{}
 
-	for _, network := range *i.Networks {
+	for idx, network := range *i.Networks {
```

Comment:
```suggestion
	for i, network := range *i.Networks {
```
nit: I think I would prefer i as that's usually used

Reply:
The only issue is that `i` is already taken for the instance variable name. I'll use `i` for the index and rename the instance variable to `instance`.

Reply:
Ah, I would prefer renaming the instance. `i` is a really short variable name for a variable with that large of a scope

Reply:
That sounds good. `instance` is already used in this scope for something else, but I'll rename the loop variable to `i` and come up with a sensible name for the instance variable currently called `i`.